src/Infrastructure/Persistence/Configurations/Participants/PriCodeEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Cfo.Cats.Domain.Entities.Participants;
+using Cfo.Cats.Infrastructure.Constants.Database;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+
+namespace Cfo.Cats.Infrastructure.Persistence.Configurations.Participants
+{
+    public class PriCodeEntityTypeConfiguration : IEntityTypeConfiguration<PriCode>
+    {
+        public void Configure(EntityTypeBuilder<PriCode> builder)
+        {
+            builder.ToTable(
+                DatabaseConstants.Tables.PriCode,
+                DatabaseConstants.Schemas.Participant);
```

Comment:
```suggestion
                DatabaseConstants.Schemas.PRI);
```
I'd suggest using the PRI schema for clarity

Reply:
changed as suggested, thanks